src/analytics/utils/index.ts

```diff
@@ -0,0 +1,12 @@
+import { Trade } from '@swapr/sdk'
+
+/**
+ * Creates an unique ID for a trade instance
+ * @param trade
+ * @returns
+ */
+export function computeTradeId(trade: Trade) {
+  return `${trade.platform.name.toLowerCase()}-${trade.chainId}/${trade.inputAmount.currency.address}-${
+    trade.outputAmount.currency.address
+  }-${trade.inputAmount.raw.toString()}-${trade.outputAmount.raw.toString()}`
+}
```

Comment:
Just to be clear, we can trace a trade with this info if we want/need, right?

Reply:
We should, ideally this should be added to `Trade` in the next iteration.